deploy/kubernetes/hcloud-csi-master.yml

```diff
@@ -69,6 +72,10 @@ rules:
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshotcontents"]
     verbs: ["get", "list"]
+  # resizer
+  - apiGroups: [""]
```

Comment:
[RBAC change in external-resizer 1.0](https://github.com/kubernetes-csi/external-resizer/blob/master/CHANGELOG/CHANGELOG-1.0.md#no-really-you-must-read-this-before-you-upgrade)